libvmi/arch/amd64.c

```diff
@@ -319,7 +319,7 @@ GSList* get_pages_ia32e(vmi_instance_t vmi, addr_t npt, page_mode_t npm, addr_t
 
                 uint64_t pgd_value = pgd_page[pgde_index];
 
-                if (ENTRY_PRESENT(vmi->os_type, pgd_value)) {
+                if (ENTRY_PRESENT(vmi->os_type != VMI_OS_UNKNOWN, pgd_value)) {
```

Comment:
This should be `VMI_OS_WINDOWS == vmi->os_type` here and all other spots. Entry present needs to know if tranisition pages are in use and those are Windows specific.

Reply:
Done